WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Custom Fields/OrderCustomFieldsDetails.swift

```diff
@@ -0,0 +1,63 @@
+import SwiftUI
+
+struct OrderCustomFieldsDetails: View {
+    @Environment(\.presentationMode) var presentationMode
+
+    let customFields: [OrderCustomFieldsViewModel]
+
+    var body: some View {
+        NavigationView {
+            GeometryReader { geometry in
+                ScrollView {
+
+                    Color(.listBackground).edgesIgnoringSafeArea(.all)
+
+                    VStack(alignment: .leading) {
+                        ForEach(customFields) { customField in
+                            TitleAndSubtitleRow(
+                                title: customField.title,
+                                subtitle: customField.content
+                            )
+                            Divider()
+                                .padding(.leading)
+                        }
+                    }
+                    .background(Color(.basicBackground))
+                }
+            }
+            .toolbar {
+                ToolbarItem(placement: .cancellationAction) {
+                    Button(action: {
+                        presentationMode.wrappedValue.dismiss()
+                    }, label: {
+                        Image(uiImage: .closeButton)
+                    })
+                }
+            }
+            .navigationTitle(Localization.title)
+            .navigationBarTitleDisplayMode(.inline)
+            .wooNavigationBarStyle()
+        }
```

Comment:
We need to set the stack navigation view style on this view:

```suggestion
        }
        .navigationViewStyle(.stack)
```

Otherwise, it uses a column view on iPads, which ends up looking like this when the modal is first opened:

![Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-07-11 at 11 31 07](https://user-images.githubusercontent.com/8658164/178246984-5fa7c798-b9bd-4e30-a9b6-a5e767ab02da.png)

Reply:
Good catch! I need to be more aware of testing on the iPad! Thanks! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/7257/commits/bd4286a1bbcf1cb16bb128947f56ba133dafcf00